### DISCUSSION TITLE
Simplest way to change color of heatmap

### DISCUSSION THREAD

**Discussion by mcp292** [vega/altair#2779] [Q&A] [answered]
# MWE
```py
import altair as alt
import numpy as np
import pandas as pd

# Compute x^2 + y^2 across a 2D grid
x, y = np.meshgrid(range(-5, 5), range(-5, 5))
z = x ** 2 + y ** 2

# Convert this grid to columnar data expected by Altair
source = pd.DataFrame({'x': x.ravel(),
                     'y': y.ravel(),
                     'z': z.ravel()})

alt.Chart(source).mark_rect().encode(
    x='x:O',
    y='y:O',
    color='z:Q'
)
```

![heatmap](https://user-images.githubusercontent.com/49123398/209883293-c08d060f-f74e-4fe8-8d6b-650e880c8079.png)

[Source](https://altair-viz.github.io/gallery/simple_heatmap.html).

# Question
What is the simplest way to change the color of this heatmap?

Is there, for example, a way to tell it I want the max color to be `"red"`, and have it interpolate from there?

# Related
The answer [provided here](https://github.com/altair-viz/altair/discussions/2605) is pretty involved.

**Comment by mattijn** [comment, marked as answer]
See the Altair docs here: https://altair-viz.github.io/user_guide/customization.html#customizing-colors
For the available color scales, see here: https://vega.github.io/vega/docs/schemes/

**Reply by mcp292** [reply]
Outstanding thank you! `color=alt.Color('z:Q', scale=alt.Scale(scheme="reds"))` did it for me!
